lhotse/features/mixer.py

```diff
@@ -41,7 +41,8 @@ def __init__(
         :param frame_shift: Required to correctly compute offset and padding during the mix.
         :param padding_value: The value used to pad the shorter features during the mix.
             This value is adequate only for log space features. For non-log space features,
-            e.g. energies, use either 0 or a small positive value like 1e-5.
+            e.g. energies, use either 0 or a small positive value like 1e-5. This value will be
+            ignore if the ``feature_extractor`` has a ``padding_value`` attribute.
```

Comment:
```suggestion
            ignored if the ``feature_extractor`` has a ``padding_value`` attribute.
```

Reply:
Should this user-provided argument have a higher priority?

Reply:
Perhaps the default value for this option can be set to None. If it is None, we use the value from the feature extractor, otherwise we use the user-provided value. But I'm not sure what should be done if they are both None. Raise a error? @pzelasko what do you think?

Reply:
I like your solution. Raising an error in this case is acceptable if you can add the default padding value attributes to the remaining feature extractors.